core/bid.py

```diff
@@ -112,7 +112,7 @@ def bid(q, api, playerList, settings):
                     for item in api.searchAuctions('player', defId=defId, max_price=bidDetails[defId]['maxBid']-subtract, start=0, page_size=50):
                         # player safety checks for every possible bid
                         # Let's look at last 5 minutes for now and bid on 5 players max
-                        if item['expires'] > 600 or bidon >= 5 or listed >= settings['maxPlayer'] or api.credits < settings['minCredits']:
+                        if item['expires'] > 300 or bidon >= 5 or listed >= settings['maxPlayer'] or api.credits < settings['minCredits']:
```

Comment:
Haha.  Math is hard ;).  If I recall correctly, I changed this on purpose once upon a time and never updated the comment.

The tradeoff here is locking up credits in a bid for a longer time vs potentially losing out on a deal.  I don't know which is best.  Have you tested it out at 5 minutes instead of 10?